src/frontend/screens/Security/index.tsx

```diff
@@ -0,0 +1,146 @@
+import { SecurityContext } from "./SecurityContext";
+import { useNavigation } from "react-navigation-hooks";
+
+const m = defineMessages({
+  title: {
+    id: "screens.Security.title",
+    defaultMessage: "Security",
+  },
+  securitySubheader: {
+    id: "screens.Security.securitySubheader",
+    defaultMessage: "Device Security",
+  },
+  passcodeHeader: {
+    id: "screens.Security.passcodeHeader",
+    defaultMessage: "App Passcode",
+  },
+  passDesriptionPassNotSet: {
+    id: "screens.Security.passDesriptionPassNotSet",
+    defaultMessage: "Passcode not set",
+  },
+  passDesriptionPassSet: {
+    id: "screens.Security.passDesriptionPassSet",
+    defaultMessage: "Passcode is set",
+  },
+  killPasscodeHeader: {
+    id: "screens.Security.killPasscodeHeader",
+    defaultMessage: "Kill Passcode",
+  },
+  killPassDescriptonPassNotSet: {
+    id: "screens.Security.killPassDescriptonPassNotSet",
+    defaultMessage: "To use, enable App Passcode",
+  },
+  killPassDescriptonPassSet: {
+    id: "screens.Security.killPassDescriptonPassSet",
+    defaultMessage: "Protect your device against seizure",
+  },
+  deviceBackup: {
+    id: "screens.Security.deviceBackup",
+    defaultMessage: "Device Backup",
+  },
+  paperKey: {
+    id: "screens.Security.paperKey",
+    defaultMessage: "Paper Key",
+  },
+  paperKeyDes: {
+    id: "screens.Security.paperKeyDes",
+    defaultMessage: "Reinstate your account if this device is lost",
+  },
+});
+
+export const Security: NavigationStackScreenComponent = () => {
+  const { passIsSet } = React.useContext(SecurityContext);
+
+  const { navigate } = useNavigation();
+
+  const [passCodeDes, killPassCodeDes] = React.useMemo(
+    () =>
+      passIsSet
+        ? [m.passDesriptionPassSet, m.killPassDescriptonPassSet]
+        : [m.passDesriptionPassNotSet, m.killPassDescriptonPassNotSet],
+    [passIsSet]
+  );
+
+  return (
+    <React.Fragment>
+      <List>
+        <ListItem button={false} style={{ marginVertical: 10 }}>
+          <ListItemText
+            style={{ textTransform: "uppercase" }}
+            primary={<FormattedMessage {...m.securitySubheader} />}
+          />
+        </ListItem>
+
+        <ListItem button={true} onPress={() => navigate("AppPasscode")}>
+          <ListItemText
+            primary={<FormattedMessage {...m.passcodeHeader} />}
+            secondary={<FormattedMessage {...passCodeDes} />}
+          />
+        </ListItem>
+
+        <ListItem
+          button={true}
+          onPress={() => {
+            return;
+          }}
+        >
+          <ListItemText
+            primary={<FormattedMessage {...m.killPasscodeHeader} />}
+            secondary={<FormattedMessage {...killPassCodeDes} />}
+          />
+        </ListItem>
+
+        <ListItem style={[styles.divder]}>
+          {/* Divider, added a React Fragment as List Item Requires a child */}
+          <React.Fragment></React.Fragment>
+        </ListItem>
```

Comment:
i'd look into adding a prop on the `ListItem` to enable a bottom/top border instead of creating a completely separate list item for a divider